replace XPU with Intel GPU (#2198)
intel/neural-compressor: examples/3.x_api/pytorch/image_recognition/torchvision_models/quantization/static_quant/ipex/README.md

   python -m pip install intel_extension_for_pytorch -f https://software.intel.com/ipex-whl-stable
   ```

### Install IPEX Intel GPU
Please build an IPEX docker container according to the [official guide](https://intel.github.io/intel-extension-for-pytorch/index.html#installation?platform=gpu&version=v2.1.30%2bIntel GPU&os=linux%2fwsl2&package=docker).

You can run a simple sanity test to double confirm if the correct version is installed, and if the software stack can get correct hardware information onboard your system. The command should return PyTorch and IPEX versions installed, as well as GPU card(s) information detected.
```bash
source {DPCPPROOT}/env/vars.sh
source {MKLROOT}/env/vars.sh
source {CCLROOT}/env/vars.sh
source {MPIROOT}/env/vars.sh

> python -c "import torch; import intel_extension_for_pytorch as ipex; print(torch.__version__); print(ipex.__version__); [print(f'[{i}]: {torch.Intel GPU.get_device_properties(i)}') for i in range(torch.Intel GPU.device_count())];"

Author: ```suggestion
python -c "import torch; import intel_extension_for_pytorch as ipex; print(torch.__version__); print(ipex.__version__); [print(f'[{i}]: {torch.xpu.get_device_properties(i)}') for i in range(torch.xpu.device_count())];"
```

```

> Please also refer to this [tutorial](https://intel.github.io/intel-extension-for-pytorch/index.html#installation?platform=gpu&version=v2.1.30%2bIntel GPU&os=linux%2fwsl2&package=conda) to check system requirements and install dependencies.

Reviewer: link issue was not fixed yet

> Please also refer to this [tutorial](https://intel.github.io/intel-extension-for-pytorch/index.html#installation?platform=gpu&version=v2.1.30%2bIntel GPU&os=linux%2fwsl2&package=conda) to check system requirements and install dependencies.

Author: Thanks

   python -m pip install intel_extension_for_pytorch -f https://software.intel.com/ipex-whl-stable
   ```

### Install IPEX Intel GPU

> Please build an IPEX docker container according to the [official guide](https://intel.github.io/intel-extension-for-pytorch/index.html#installation?platform=gpu&version=v2.1.30%2bIntel GPU&os=linux%2fwsl2&package=docker).

Author: ```suggestion
Please build an IPEX docker container according to the [official guide](https://intel.github.io/intel-extension-for-pytorch/index.html#installation?platform=gpu&version=v2.1.30%2bxpu&os=linux%2fwsl2&package=docker).
```

source {MPIROOT}/env/vars.sh
python -c "import torch; import intel_extension_for_pytorch as ipex; print(torch.__version__); print(ipex.__version__); [print(f'[{i}]: {torch.xpu.get_device_properties(i)}') for i in range(torch.xpu.device_count())];"
```

> Please also refer to this [tutorial](https://intel.github.io/intel-extension-for-pytorch/index.html#installation?platform=gpu&version=v2.1.30%2bIntel GPU&os=linux%2fwsl2&package=conda) to check system requirements and install dependencies.

Author: ```suggestion
Please also refer to this [tutorial](https://intel.github.io/intel-extension-for-pytorch/index.html#installation?platform=gpu&version=v2.1.30%2bxpu&os=linux%2fwsl2&package=conda) to check system requirements and install dependencies.
```

bash run_benchmark.sh --input_model=resnext101_32x16d_wsl --dataset_location=/path/to/imagenet --mode=performance/accuracy --int8=true/false
```

# Run with Intel GPU

> Note: All torchvision model names can be passed as long as they are included in `torchvision.models`, below are some examples.

### 1. ResNet18 With Intel PyTorch Extension

```shell

> python main.py -t -a resnet18 --ipex --pretrained /path/to/imagenet --Intel GPU

Author: ```suggestion
python main.py -t -a resnet18 --ipex --pretrained /path/to/imagenet --xpu
```

bash run_benchmark.sh --input_model=resnext101_32x16d_wsl --dataset_location=/path/to/imagenet --mode=performance/accuracy --int8=true/false
```

# Run with Intel GPU

> Note: All torchvision model names can be passed as long as they are included in `torchvision.models`, below are some examples.

### 1. ResNet18 With Intel PyTorch Extension

```shell

> python main.py -t -a resnet18 --ipex --pretrained /path/to/imagenet --Intel GPU

Reviewer: , if script does not change, should not replace in cmdline

```
or
```shell
bash run_quant.sh --input_model=resnet18 --dataset_location=/path/to/imagenet

> bash run_benchmark.sh --input_model=resnet18 --dataset_location=/path/to/imagenet --mode=performance/accuracy --int8=true/false --Intel GPU=true/false

Author: ```suggestion
bash run_benchmark.sh --input_model=resnet18 --dataset_location=/path/to/imagenet --mode=performance/accuracy --int8=true/false --xpu=true/false
```

> bash run_benchmark.sh --input_model=resnet18 --dataset_location=/path/to/imagenet --mode=performance/accuracy --int8=true/false --Intel GPU=true/false

Reviewer: same as above
